db/migrate/20220324163932_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.text :name
+      t.text :photo
```

Comment:
- [ ] As per the previous review, you have to change the data type to `text` instead of `string` only for **bio** column. Since the bio column can have large text. Keep the `name` and `photo` columns `string` data type.

Hint:
```ruby
create_table :users do |t|
    t.string :name
    t.string :photo # as photo should be a link to an image not a file
    t.text :bio
    t.integer :posts_counter
    t.timestamps # this will crate uopdated_at and created_at columns
  end
```